demo/rviz.launch.py

```diff
@@ -0,0 +1,60 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node, PushRosNamespace
+from launch.actions import DeclareLaunchArgument, OpaqueFunction
+from launch.substitutions import EnvironmentVariable, LaunchConfiguration
+
+def replace_rviz_fixed_frame(input_file, output_file, fix_frame):
+    with open(input_file, 'r') as file:
```

Comment:
You can use `ReplaceString` from nav2 like here:
https://github.com/husarion/rosbot_ros/blob/add-gazebo-namespaces/rosbot_gazebo/launch/spawn.launch.py#L64

Reply:
I need to install nav2 package inside astra docker which doesn't seems right for me